test/cmd/builds.sh

```diff
@@ -75,6 +75,20 @@ os::cmd::expect_success_and_text 'oc get bc/centos -o=jsonpath="{.spec.output.to
 # Ensure output is valid JSON
 os::cmd::expect_success 'oc new-build -D "FROM centos:7" -o json | python -m json.tool'
 
+# Ensure post commit hook is executed
+os::cmd::expect_success 'oc new-build -D "FROM busybox:1"'
+os::cmd::try_until_text 'oc get istag busybox:1' 'busybox@sha256:'
+os::cmd::expect_success 'oc patch bc/busybox -p '\''{"spec":{"postCommit":{"script":"echo hello $1","args":["world"],"command":null}}}'\'
+os::cmd::expect_success_and_text 'oc get bc/busybox -o=jsonpath="{.spec.postCommit['\''script'\'','\''args'\'','\''command'\'']}"' '^echo hello \$1 \[world\] \[\]$'
```

Comment:
This test makes sure we persist the post commit hook correctly.

The line below is a test that ensures the hook is actually executed as part of the build, pending a fix to #7174.